libs/react/src/lib/modal/modal.tsx

```diff
@@ -1,5 +1,5 @@
 import { ModalType, Size } from '@sebgroup/extract'
-import { MouseEvent, ReactNode } from 'react'
+import { MouseEvent, ReactNode, memo } from 'react'
```

Comment:
I've seen you guys don't use memo, how come?

Reply:
From my own experience I've always used it as a best practice, even if some components don't need it or if the actual performance difference isn't that big. But if you start using React.memo it's also important to use React.useCallback, otherwise the component is still going to rerender every time due to the function passed to the component changing on every rerender.

https://stackoverflow.com/questions/53074551/when-should-you-not-use-react-memo

Reply:
Thank you. I also looked around a bit more, and based on [this piece of advice](https://react.dev/reference/react/memo#should-you-add-memo-everywhere) in the React docs, I think we should skip using memo by default and only add it if some component has measurable performance issues.

Basically it has little to no benefit in most cases, but can potentially make troubleshooting other issues more difficult. Locally in an application, this is probably almost never an issue, but in Greens case I think we should favor ease of maintenance unless there are clear benefits to adding something.

Reply:
Sure, the one importing the component can always wrap it in memo if needed. I've myself not experienced any problems when troubleshooting with memo, but I agree that the code becomes a bit more complex maybe. The times I've really needed it has been with huge lists of data. The only thing I feel is a bit ugly from what I've seen from SEB is that in some projects updating one input rerenders the entire page since the main state is loaded in the main component. Therefore I feel it's good to at least know about memo. But I removed it from this component now :)

Reply:
Sure, the one importing the component can always wrap it in memo if needed. I've myself not experienced any problems when troubleshooting with memo, but I agree that the code becomes a bit more complex maybe. The times I've really needed it has been with huge lists of data. The only thing I feel is a bit ugly from what I've seen from SEB is that in some projects updating one input rerenders the entire page since the main state is loaded in the main component. Therefore I feel it's good to at least know about memo. But I removed it from this component now :)